receiver/filereceiver/README.md

```diff
@@ -0,0 +1,27 @@
+This receiver will read pipeline data from JSON files. The data is written in
+[Protobuf JSON
+encoding](https://developers.google.com/protocol-buffers/docs/proto3#json)
+using [OpenTelemetry
+protocol](https://github.com/open-telemetry/opentelemetry-proto).
+
+The receiver will watch the directory and read files. If a file is updated or added,
+the receiver will read it in its entirety again.
+
+Please note that there is no guarantee that exact field names will remain stable.
+This intended for primarily for debugging Collector without setting up backends.
+
+Supported pipeline types: traces, metrics, logs
+
+## Getting Started
+
+The following settings are required:
+
+- `path` (no default): folder to watch.
```

Comment:
This seems a bit limited. A recursive glob pattern like file log receiver might serve more scenarios.
WDYT ?

Reply:
See other comment - this is because this is POC, it needs love and care to come up in maturity.